Northstar.Client/mod/scripts/vscripts/ui/menu_ns_connect_password.nut

```diff
@@ -56,6 +56,6 @@ void function ConnectWithPassword( var button )
 	if ( GetTopNonDialogMenu() == file.menu )
 	{
 		TriggerConnectToServerCallbacks()
-		thread ThreadedAuthAndConnectToServer( Hud_GetUTF8Text( Hud_GetChild( file.menu, "EnterPasswordBox" ) ) )
+		thread ThreadedAuthAndConnectToServer( Hud_GetUTF8Text( Hud_GetChild( file.menu, "EnterPasswordBox" ) ), true )
```

Comment:
This should be changed later, moving mod downloading to after password and such

Reply:
Moved to #781